src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -119,14 +147,59 @@ private void openEditor(@Nonnull Player p, @Nonnull Block projector) {
             return false;
         });
 
-        menu.addItem(1, new CustomItemStack(Material.CLOCK, "&7Offset: &e" + NumberUtils.roundDecimalNumber(Double.valueOf(BlockStorage.getLocationInfo(projector.getLocation(), OFFSET_PARAMETER)) + 1.0D), "", "&fLeft Click: &7+0.1", "&fRight Click: &7-0.1"));
+        menu.addItem(1, new CustomItemStack(Material.CLOCK, "&7Offset X: &e" + getXOffset(projector), "", "&fLeft Click: &7+0.1", "&fRight Click: &7-0.1"));
+        menu.addMenuClickHandler(1, (pl, slot, item, action) -> {
+            double offsetX = NumberUtils.reparseDouble(getXOffset(projector) + (action.isRightClicked() ? -0.1F : 0.1F));
+            double offsetY = getYOffset(projector);
+            double offsetZ = getZOffset(projector);
+
+            if (offsetX < -MAX_HORIZONTAL_OFFSET) {
+                offsetX = -MAX_HORIZONTAL_OFFSET;
+            } else if (offsetX > MAX_HORIZONTAL_OFFSET) {
+                offsetX = MAX_HORIZONTAL_OFFSET;
+            }
```

Comment:
Math.max and Math.min should be used here, there might also be a method to do both